src/main/helpers/fs.helpers.ts

```diff
@@ -52,16 +52,29 @@ export async function getFoldersInFolder(folderPath: string, opts?: {ignoreSymli
     return (await Promise.all(promises)).filter(folder => folder);
 }
 
-export function moveFolderContent(src: string, dest: string): Observable<Progression>{    
-    const progress: Progression = { current: 0, total: 0 }; 
+export async function getFilesInFolder(folderPath: string): Promise<string[]> {
+    if(!(await pathExist(folderPath))){ return []; }
+
+    const files = await readdir(folderPath, {withFileTypes: true});
+
+    const promises = files.map(async file => {
+        if(file.isFile()){ return path.join(folderPath, file.name); }
```

Comment:
The function in map don't need to be async

Reply:
Without async, you can use the 'reduce' method to avoid two full iterations over the array by removing the filter at the end 😉

Reply:
I copied this from FoldersInFolders, just with a flipped condition and no symlink handling.
But I guess I don't need the async function that was used there.

Would be really nice to add eslint, prettier, and maybe sonar to CI so computers can find these issues instead of you or I.
But I guess that will be it's own PR.

Reply:
Fixed.